lib/init/index.js

```diff
@@ -1,65 +1,152 @@
 "use strict";
 
-const yeoman = require("yeoman-environment");
-const Generator = require("yeoman-generator");
 const path = require("path");
-const defaultGenerator = require("../generators/init-generator");
-const runTransform = require("./transformations/index");
+const j = require("jscodeshift");
+const chalk = require("chalk");
+const pEachSeries = require("p-each-series");
+
+const runPrettier = require("../utils/run-prettier");
+
+const entryTransform = require("./transformations/entry/entry");
+const outputTransform = require("./transformations/output/output");
+const contextTransform = require("./transformations/context/context");
+const resolveTransform = require("./transformations/resolve/resolve");
+const devtoolTransform = require("./transformations/devtool/devtool");
+const targetTransform = require("./transformations/target/target");
+const watchTransform = require("./transformations/watch/watch");
+const watchOptionsTransform = require("./transformations/watch/watchOptions");
+const externalsTransform = require("./transformations/externals/externals");
+const nodeTransform = require("./transformations/node/node");
+const performanceTransform = require("./transformations/performance/performance");
+const statsTransform = require("./transformations/stats/stats");
+const amdTransform = require("./transformations/other/amd");
+const bailTransform = require("./transformations/other/bail");
+const cacheTransform = require("./transformations/other/cache");
+const profileTransform = require("./transformations/other/profile");
+const mergeTransform = require("./transformations/other/merge");
+const parallelismTransform = require("./transformations/other/parallelism");
+const recordsInputPathTransform = require("./transformations/other/recordsInputPath");
+const recordsOutputPathTransform = require("./transformations/other/recordsOutputPath");
+const recordsPathTransform = require("./transformations/other/recordsPath");
+const moduleTransform = require("./transformations/module/module");
+const pluginsTransform = require("./transformations/plugins/plugins");
+const topScopeTransform = require("./transformations/top-scope/top-scope");
+const devServerTransform = require("./transformations/devServer/devServer");
+const modeTransform = require("./transformations/mode/mode");
+const resolveLoaderTransform = require("./transformations/resolveLoader/resolveLoader");
 
 /**
  *
- * Runs yeoman and runs the transformations based on the object
- * built up from an author/user
+ * Runs the transformations from an object we get from yeoman
```

Comment:
The comment should be moved after the `transformObject`

Reply:
Thanks